gnovm/pkg/gnolang/values_string.go

```diff
@@ -227,11 +227,29 @@ func (tv *TypedValue) Sprint(m *Machine) string {
 	case *ArrayType:
 		return tv.V.(*ArrayValue).String()
 	case *SliceType:
-		return tv.V.(*SliceValue).String()
+		switch sv := tv.V.(type) {
+		case nil:
+			return tv.String()
+		case *SliceValue:
+			return sv.String()
+		default:
+			panic(fmt.Sprintf(
+				"unexpected slice type %v",
+				reflect.TypeOf(tv.V)))
```

Comment:
for this and the one below, why not this?

```suggestion
				"unexpected slice type %T",
				tv.V))
```

Reply:
neat. thanks.